Setup/InstallData.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace Klaviyo\Reclaim\Setup;
+  
+use \Magento\Framework\Setup\InstallDataInterface;
+use \Magento\Framework\Setup\ModuleDataSetupInterface;
+use \Magento\Framework\Setup\ModuleContextInterface;
+use \Magento\Authorization\Model\Acl\Role\Group as RoleGroup;
```

Comment:
why the as here?

Reply:
makes it easier to read IMO, otherwise it would just be Group